htgr/htr-pm/ss-main.i

```diff
@@ -186,12 +186,12 @@ rho_in            = 6.3306  # Helium density at 7 MPa and 523.15 K (from NIST)
   # Velocity
   [vel_x]
     scaling = 1e-3
-    initial_condition = 1.0E-08
+    initial_condition = 1 # 1.0E-08
     block = ${fluid_blocks}
   []
   [vel_y]
     scaling = 1e-3
-    initial_condition = 1.0E-08
+    initial_condition = 1 # 1.0E-08
```

Comment:
this seems like a very significant change. What was the motivation?

Reply:
it s closer to the solution, see the gold files